### PR TITLE
Database

### PR DESCRIPTION
**Pull request by ardaboyadzhiev**
Added Required and MaxLength() attributes to the string fields. Database updated successfully.

### CODE REVIEW

**Comment by georgipetkov18** [comment]
Do not be gei and everything will be alright.

**Comment by apriori-yarik** [comment]
Modify README.md, Message.Content - without maxlength

File: OMG.Classroom/DataAccessLayer/Entities/Message.cs [hunk 1]
```diff
@@ -8,6 +9,8 @@ namespace DataAccessLayer.Entities
 {
     public class Message : BaseEntity
     {
+        [Required]
+        [MaxLength(255)]
```
Comment (apriori-yarik) [inline OMG.Classroom/DataAccessLayer/Entities/Message.cs]: 255 could be not enough for a detailed question/answer. Just remove it

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-# omg-classroom
\ No newline at end of file
+az sum gei
```
Comment (apriori-yarik) [inline README.md]: LGBTQ is not supported in this project, leave it as it was

**Review by apriori-yarik — CHANGES_REQUESTED** [review]

**Pull request merged**